package/lighthouse-crd.yaml

```diff
@@ -1,12 +1,12 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: serviceexports.lighthouse.submariner.io
+  name: serviceimports.lighthouse.submariner.io
 spec:
   group: lighthouse.submariner.io
   version: v2alpha1
   names:
-    kind: ServiceExport
-    plural: serviceexports
-    singular: serviceexport
+    kind: ServiceImport
+    plural: serviceimports
+    singular: serviceimport
```

Comment:
Why are you replacing ServiceExport here?

Reply:
> Why are you replacing ServiceExport here?

This was stop-gap CRD used in e2e before helm+operator can install ServiceExport. Since they do, currently this is not being used. With ServiceImport, we will use this to install ServiceImport till changes are merged in helm+operator and then remove this file altogether.